--- a/autotests/testincidenceformatter.cpp
+++ b/autotests/testincidenceformatter.cpp
@@ -50,6 +50,8 @@
     QStandardPaths::setTestModeEnabled(true);
     GrantleeTemplateManager::instance()->setTemplatePath(QStringLiteral(TEST_TEMPLATE_PATH));
     GrantleeTemplateManager::instance()->setPluginPath(QStringLiteral(TEST_PLUGIN_PATH));
+    // This test only works using the oxygen icon theme
+    QIcon::setThemeName(QStringLiteral("oxygen"));
 }
 
 void IncidenceFormatterTest::testRecurrenceString()
